internal/qaengine/engine.go

```diff
@@ -175,3 +175,90 @@ func changeSelectToInputForOther(prob qatypes.Problem) qatypes.Problem {
 	}
 	return prob
 }
+
+// Convenience functions
+
+// FetchStringAnswer asks a input type question and gets a string as the answer
```

Comment:
Let's add one for multi line too. We don't use it now, but when users want it, they need not hunt and find out how to add it.

Reply:
Actually I think we should remove MultiLine. Doesn't seem that useful.

Reply:
done